requirements/py38.yml

```diff
@@ -12,7 +12,7 @@ dependencies:
 
 # core dependencies
   - antlr-python-runtime 4.7.2.*
-  - cftime>=1.2
+  - cftime>=1.2,<1.5.1.1
```

Comment:
We probably don't want this pin but I'm not sure what is the right fix for cftime new default. Should I just change the expected calendar in that test? If so, the test is a bit flaky and ensuring the expected calendar explicitly is probably better than relaying on cftime's defaults.

Reply:
Over at #205, @larsbarring actively wants to change the expected calendar in that test.  But now I’m confused, as I thought https://github.com/Unidata/cftime/pull/257 would be needed to change it, and `cftime`‘s GitHub page is not indicating any releases since September 😕

Reply:
I was in contact with cftime developers in [Unidata/cdtime#256](https://github.com/Unidata/cftime/issues/256) and initially they did a required change in [Unidata/cdtime#257](https://github.com/Unidata/cftime/pull/257). But it it broke xarray, and thus was not merged into main. So I [asked the  xarray devs](https://github.com/pydata/xarray/discussions/6162) and they have some ideas. But I am already way out of my competence zone, so I will just have to wait see --- and hope that I have not caused any trouble for you  guys.

Reply:
> hope that I have not caused any trouble for you guys.

Not at all, I think by opening #205 you have helped us prepare for this change whereas usually we catch up some time later.

I am just confused because there seems to be a new `cftime` release that is not advertised on their GitHub page (which is where I had been checking).

Reply:
@rcomer maybe we can take the conservative approach for now and issue one more release with a cftime pin to unlock a  py310 wheel?